modules/search.py

```diff
@@ -10,31 +10,30 @@
 import re
 import web
 
-def google_ajax(query): 
-    """Search using AjaxSearch, and return its JSON."""
-    if isinstance(query, str): 
-        query = query.encode('utf-8')
-    uri = 'https://ajax.googleapis.com/ajax/services/search/web'
-    args = '?v=1.0&safe=off&q=' + web.quote(query)
-    bytes = web.get(uri + args, headers={'Referer': 'https://github.com/sbp/phenny'})
-    return web.json(bytes)
+
+r_google = re.compile(r'href="\/url\?q=(http.*?)&')
 
 def google_search(query): 
-    results = google_ajax(query)
-    try: return results['responseData']['results'][0]['unescapedUrl']
-    except IndexError: return None
-    except TypeError: 
-        print(results)
-        return False
+    query = web.quote(query)
+    uri = 'https://google.co.uk/search?q=%s' % query
```

Comment:
I'm surprised this works and Google isn't blocking it. Any reason you're using google.co.uk over google.com?